### PR TITLE
PS-4641 innodb_flush_method is not clear enough

### PR DESCRIPTION
**Pull request by patrickbirch**
updated innodb_io.rst with the suggested changes

### CODE REVIEW

**Review by dutow — CHANGES_REQUESTED** [review]
There are no "doublewrite files". Regular doublewrite is part of the system tablespace. (probably a "parallel" missing at two places?)

Also, I think parallel doublewrite uses a single file, not multiple files, I'm not sure if we should use plural.

**Comment by patrickbirch** [comment]
Removed the plural from "doublewrite".

**Comment by patrickbirch** [comment]
waiting for comments or approval

File: doc/source/scalability/innodb_io.rst
Comment (inikep) [inline doc/source/scalability/innodb_io.rst]:
```suggestion
    use O_DIRECT to open the data files and parallel doublewrite files, but does not
```

File: doc/source/scalability/innodb_io.rst
Comment (inikep) [inline doc/source/scalability/innodb_io.rst]:
```suggestion
    parallel doublewrite files. This option isn't suitable for *XFS* file system.
```

File: doc/source/scalability/innodb_io.rst
Comment (inikep) [inline doc/source/scalability/innodb_io.rst]:
```suggestion
    use O_DIRECT to open data files, log files, and parallel doublewrite files
```

File: doc/source/scalability/innodb_io.rst
Comment (inikep) [inline doc/source/scalability/innodb_io.rst]:
```suggestion
    parallel doublewrite files. This option is recommended when |InnoDB| log files are big (more than 8GB), otherwise there might be even a performance degradation. **Note**: When using this option on *ext4* filesystem variable :variable:`innodb_log_block_size` should be set to 4096 (default log-block-size in *ext4*) in order to avoid the ``unaligned AIO/DIO`` warnings.
```

**Review by inikep — CHANGES_REQUESTED** [review]

File: doc/source/scalability/innodb_io.rst
Comment (patrickbirch) [reply doc/source/scalability/innodb_io.rst]: Updated.

File: doc/source/scalability/innodb_io.rst
Comment (patrickbirch) [reply doc/source/scalability/innodb_io.rst]: updated.

File: doc/source/scalability/innodb_io.rst
Comment (patrickbirch) [reply doc/source/scalability/innodb_io.rst]: updated.

File: doc/source/scalability/innodb_io.rst
Comment (patrickbirch) [reply doc/source/scalability/innodb_io.rst]: updated.

**Review by inikep — APPROVED** [review]
LGTM

**Pull request merged**
